observability/logging/logging-6.0/log6x-clf.adoc

```diff
@@ -0,0 +1,79 @@
+:_mod-docs-content-type: ASSEMBLY
+include::_attributes/common-attributes.adoc[]
+[id="log6x-CLF_{context}"]
+= Configuring log forwarding
+:context: logging-6x
+
+toc::[]
+
+The `ClusterLogForwarder` CR serves as a single point of configuration for log forwarding, making it easier to manage and maintain log collection and forwarding rules.
+
+* Defines inputs (sources) for log collection
+* Specifies outputs (destinations) for log forwarding
+* Configures filters for log processing
+* Defines pipelines to route logs from inputs to outputs
+* Indicates management state (managed or unmanaged)
+
+include::modules/logging-otlp-forward.adoc[leveloffset=+1]
+
+////
+= OTLP Output
+
+The OTLP output forwards logs using HTTP or JSON as defined by the OpenTelemetry Observability Framework. This is a configuration guide for the `ClusterLogForwarder` spec introduced to send logs to OTLP receivers.
+
+*OTLP* describes the *protocol* for encoding, transporting, and delivering telemetry data between sources using the https://opentelemetry.io/docs/specs/otlp/[OpenTelemetry OTLP Specification]
+
+.Technical Preview
+This feature is currently in tech-preview and an annotation is required
+
+`observability.openshift.io/tech-preview-otlp-output: "enabled"`
+
+== Configuring the Forwarder
+A new output type `otlp` has been defined in the `ClusterLogForwarder`
+
+.ClusterLogForwarder
+[source,yaml]
+----
+apiVersion: observability.openshift.io/v1
+kind: ClusterLogForwarder
+metadata:
+    name: my-logforwarder
+    namespace: my-app-namespace
+    annotations:
+        observability.openshift.io/tech-preview-otlp-output: "enabled"
+spec:
+    outputs:
+    - type: otlp <1>
+      name: otel-collector
+      tls:
+        insecureSkipVerify: true <2>
+      otlp:
+        url: 'http://my-otel-receiver-service:4318/v1/logs' <3>
+        tuning:
+            compression: none <4>
+        authentication:
+            token:
+            from: serviceAccount <5>
+    pipelines:
+    - name: my-pipeline
+        inputRefs:
+        - application
+        - infrastructure
+        outputRefs:
+        - otel-collector
+    serviceAccount:
+    name: logger-admin
+----
+<1> Output `type` is '*otlp*'
+<2> `tls` includes the standard certificate configuration or specify `insecureSkipVerify`: '*true*'
+<3> `otlp` `url` specify a valid host:port of the otel receiver and *MUST* terminate with "*/v1/logs*"
+<4> `otlp` `tuning` `compression` default value is "*gzip*". Specify "*none*" to turn compression off.
+<5> `otlp` `authentication` is optional and specifies a `token` `from` and a value of "*serviceAccount*"
+.. The token can also be read from a secret
+.. Also available with `username` and `password` authentication spec (refer to HTTP Auth Specification for full scope)
+
+[NOTE]
+====
+TLS InsecureSkipVerify option is not recommended for production configurations. If true, the client will be configured to skip validating server certificates.
+====
+////
```

Comment:
I'm assuming this is here for ease of reference creating this PR, given the comment blocks.